packages/frontend-api/src/Model/Complaint/ComplaintApiFacade.php

```diff
@@ -194,6 +195,22 @@ public function getCustomerUserComplaintsLimitedList(
         return $this->complaintRepository->getCustomerUserComplaintsLimitedList($customerUser, $limit, $offset, $search);
     }
 
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Customer\Customer $customer
+     * @param int $limit
+     * @param int $offset
+     * @param string|null $search
+     * @return array
```

Comment:
```suggestion
     * @return \Shopsys\FrameworkBundle\Model\Complaint\Complaint[]
```

Could you fix that for `getCustomerUserComplaintsLimitedList` as well, please?

Reply:
Could you fix that for `getCustomerUserComplaintsLimitedList` as well, please?